docs/content/getting-started/setup-openfga/docker-setup.mdx

```diff
@@ -480,7 +480,7 @@ The following table enumerates the experimental flags, a description of what the
 |-------------------|--------------------------------------------------------------------|-----------------------|----------------------------|
 | otel-metrics      | Enables support for exposing OpenFGA metrics through OpenTelemetry | `0.3.2 <= v < v0.3.5` | became default in `v0.3.5` |
 | list-objects      | Enables ListObjects API                                            | `0.2.0 <= v < v0.3.3` | became default in `v0.3.3` |
-| check-query-cache | Enables caching of check subproblem result                         | `1.3.1 <= v `         | currently experimental     |
+| check-query-cache | Enables caching of check subproblem result                         | `1.3.1 <= v < v1.3.6` | became default in `v1.3.6` |
```

Comment:
Default is not enabled. The only thing changed is the experimental flag was removed.

Reply:
reworded